resources/js/functions.js

```diff
@@ -392,15 +392,55 @@ function getOS() {
   return osname;
 }
 
+function toggleElement(evt, elemId) {
```

Comment:
I don’t have a strong opinion here, but isn’t it risky to introduce several top-level names like this one? The risk I see is name clashes, or cross-site code injections that would override the meaning of those names.

Reply:
is the suggestion here to make less generic names? - I was mostly following the convention for the whole js file

Reply:
so next step would be to instead of use the `onclick` attribute in html, add event listeners when the document loads

Reply:
done